src/components/AccordionComponent/Accordion.tsx

```diff
@@ -0,0 +1,29 @@
+import React, { ReactNode, useState } from 'react';
+import ClassTrackerItems from './ClassTrackerItems';
+
+type ClassTrackerItemsType = {
+  title: string;
+  content: ReactNode;
+};
+
+const Accordion = ({ items }: { items: Array<ClassTrackerItemsType> }) => {
+  //   const [isExpanded, setIsExpanded] = useState<Boolean>(false);
+  const [currentID, setCurrentID] = useState<number>();
+
```

Comment:
for ID, in Mongo its ObjectID (a BSON datatype) and it looks like this
```
6266ef8a85e22dc81f841366
```
So I suggest to use string here instead of number.

Reply:
Makes sense, I will update that!